test/integration/channel.ts

```diff
@@ -768,7 +766,7 @@ describe('Channel', function () {
       poi,
       payload: signedTx
     })
-    const closeSoloTxFee = unpackTx(closeSoloTx).tx.fee
+    const closeSoloTxFee = unpackTx<TxChannelCloseSolo>(closeSoloTx).tx.fee
```

Comment:
```suggestion
    const closeSoloTxFee = unpackTx(closeSoloTx, TX_TYPE.channelCloseSolo).tx.fee
```
So, what about doing a runtime check? And switching the return type accordingly?

Reply:
Currently working on this, since unpackTx already accepts an optional argument `fromRlpBinary`, should I create an object argument regarding options, something like `options: {
txType?: TxType,
fromRlpBinary?: boolean
}?`